crates/core/src/lattice.rs

```diff
@@ -0,0 +1,282 @@
+//! Core reusable functionality related to wasmCloud lattices
+
+use core::fmt;
+
+use anyhow::{anyhow, bail, ensure, Context};
+use nkeys::{KeyPair, KeyPairType};
+use serde::{Deserialize, Serialize};
+use sha2::{Digest, Sha256};
+use ulid::Ulid;
+use uuid::Uuid;
+use wascap::jwt;
+use wascap::prelude::Claims;
+
+use crate::otel::TraceContext;
+use crate::wit::{deserialize_wit_map, serialize_wit_map};
+
+/// Key of an issuer (nkey)
+pub type ClusterIssuerKey = String;
+
+/// List of (allowed) cluster issuers
+pub type ClusterIssuers = Vec<ClusterIssuerKey>;
+
```

Comment:
After comments in https://github.com/wasmCloud/wasmCloud/pull/1575, we generally had consensus that this type of type aliasing was a higher burden to mentally parse, like when the alias is just for a String or just for a Vec / HashMap. What are your thoughts here?

Reply:
I think this kind of aliasing is definitely unnecessary -- `Vec<ClusterIssuerKey>` is definitely fine with me -- I didn't add this particular one.

Reply:
removed!